src/main/java/org/fundacionjala/sfdc/pageobjects/Accounts/SFANewModifyPage.java

```diff
@@ -0,0 +1,119 @@
+package org.fundacionjala.sfdc.pageobjects.Accounts;
+
+import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+/**
+ * SFANewModifyPage.java
+ * Class that represents the form where you fill fields in order to create or modify an account.
+ */
+public class SFANewModifyPage extends SalesForceConnection{
+    private By accountNameTextField;
+    private By parentAccountTextField;
+    private By accountNumberTextField;
+    private By accountSiteTextField;
+    private By accountTypeList;
+    private By typeList;
+    private By industryList;
+    private By annualRevenueTextField;
+    private By ratingList;
+    private By phoneTextField;
+    private By faxTextField;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "faxTextField" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)